pages/en/blog/announcements/diving-into-the-nodejs-website-redesign.md

```diff
@@ -0,0 +1,118 @@
+---
+date: '2024-03-24T12:30:09.000Z'
+category: announcements
+title: Diving into the Node.js Website Redesign
+layout: blog-post
+author: Brian Muenzenmeyer
+---
+
+By now you've noticed nodejs.org's fresh new look!
+
+We've taken great care in approaching this design with a nod to the past and look to the future. The site has many converging use cases, thousands of pages, and is a daily resource to many. The whole story had some dead ends and detours. But in the end it was a collective effort; coming to life with the contributions of over three dozen contributors and fantastic teamwork with select partners. The site improves the information architecture, brings content to users' fingertips like never before, and puts in place a stable development platform for years to come. We've also iterated on a revamped developer experience, clearer CI/CD feedback, and an approachable tech stack.
+
+Read on for a deeper dive into the journey and insights into what's to come.
+
+![A screenshot of the nodejs.org website from 2024.](/static/images/blog/announcements/2024-nodejs-screenshot.png)
+
+## Scale and Constraints
+
+The nodejs.org site has been around for over 14 years. A design resembling the utilitarian download and docs homepage first appeared in late 2011. This was Node.js `0.6` days, for context.
+
+![A screenshot of the nodejs.org website from late 2011.](/static/images/blog/announcements/2011-nodejs-screenshot.png)
+
+Ever since, the site has slowly grown in scale, as the project has needed it to. It contains 1600+ pages. At its peak it had almost 20 localized languages. The domain serves _3 billion_ requests a month, with _2 petabytes_ of data transferred. It would be an understatement to say that the site is a critical resource for the Node.js community. It was absolutely paramount to ensure changes didn't cause unnecessary disruption, both to users and contributors.
+
+## A False Start: nodejs.dev
+
```

Comment:
maybe we should put a link to the repo somewhere for those who are curious

Reply:
added via https://github.com/nodejs/nodejs.org/pull/6524/commits/813905a010a94c696d2275f4ddaea9c42f2099dc - good idea!